src/wizard/index.tsx

```diff
@@ -89,9 +88,10 @@ export default function Wizard({
     );
   }
 
-  useEffectOnUpdate(() => {
+  useEffect(() => {
     trackStartStep(actualActiveStepIndex);
-  }, [actualActiveStepIndex, trackStartStep]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This disable can be avoided. `useWizardAnalytics` is not a hook, because it does not contain any first-class react hooks. These functions can be imported directly

```
import { trackStartStep } from './internal/analytics';
```

and this will make linter happy without workarounds

Reply:
Agreed. I think I had bigger plans to have it in a neat single hook but you're right that it's better to just import the functions. Will change in next commit.